<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crear Experimento</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Parámetros:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cantidad:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCrear">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Duración:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTipoDuracion">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Minutos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Horas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dias</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Semanas</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frecuencia:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTipoFrecuencia">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Minutos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Horas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dias</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Semanas</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEditNombre">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>331</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCantidad">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>331</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDuracion">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFrecuencia">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParams">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>331</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
